{"uuid": "baa76688-d6ff-4df9-9073-29d55e299a8f", "children": ["dc98fd02-531a-4a24-b192-0852ba31c1cf"], "befores": [{"name": "Conftest", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'get'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 923, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 783, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/liuchengxu/work/sunxing/sunxing/TestSuite/CcsUserLoginController/conftest.py\", line 26, in Conftest\n    token = response.get('msg')\n"}, "start": 1582534319997, "stop": 1582534320043}], "start": 1582534319997, "stop": 1582534320105}